deploy/config

```diff
@@ -158,8 +158,8 @@ es_etc_node_datagroups['es']:
 es_api_host: "{{inventory_hostname}}"
 es_heap_size: 1500m
 es_etc_discovery_zen_minimum_master_nodes: 1
-es_snapshot_host: "{{ groups['es'][0] }}"
```

Comment:
@msknext why did you delete the spaces, as it's the best idiomatic ansible way to define variables.

Reply:
@rjshrjndrn  Yes I agree but two reasons:
1. To make it uniform with "{{inventory_hostname}}"
2. When doing an awk without -F ":", this will give the output as "{{.  As I saw in your original script you had used awk without -F ":", I used the same logic in my validate script to retrieve values into the array.
But it can be changed back . Please let me know.